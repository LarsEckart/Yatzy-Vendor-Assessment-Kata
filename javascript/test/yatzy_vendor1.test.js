
const valid_categories = require('../src/yatzy_vendor1');


test('all categories', () => {
    const categories = valid_categories();
    expect(categories).toMatchSnapshot();
});
