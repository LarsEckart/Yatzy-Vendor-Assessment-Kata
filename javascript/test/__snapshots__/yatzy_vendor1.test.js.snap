// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`all categories 1`] = `
Array [
  "chance",
  "yatzy",
  "ones",
  "twos",
  "threes",
  "fours",
  "fives",
  "sixes",
  "pair",
  "threeofakind",
  "fourofakind",
  "smallstraight",
  "largestraight",
  "twopairs",
  "fullhouse",
]
`;
